it/DeleteChildLinkFromLegalUnitSpec.scala

```diff
@@ -0,0 +1,149 @@
+        patch(s"""[{"op": "test", "path": "/children/$VatRef", "value": "$VatUnitAcronym"},
+                  |{"op": "remove", "path": "/children/$VatRef"}]""".stripMargin))
+
+      Then("a Conflict response is returned")
+      response.status shouldBe CONFLICT
+    }
+
+    /*
+     * The target row exists, but the column we would like to delete does not.
+     * To make our operation idempotent (and thus safe to retry) we treat this as a success case.
+     *
+     * Note that this was probably a mistake.  While this is the correct semantics for a delete, and we require
+     * this for the sequence of actions involved in moving an admin unit to be re-runnable on failure, this specific
+     * client request is a patch, and a patch should fail if a test condition is not satisfied.
+     */
+    scenario("when the link does not exist") { wsClient =>
+      Given(s"there exists a Legal Unit identified by UBRN $TargetUBRN having no child link to the VAT unit with reference $VatRef")
+      stubHBaseFor(aUnitLinksExactRowKeyRequest(withUnitId = TargetLegalUnitId, withUnitType = LegalUnit, withPeriod = RegisterPeriod).
+        willReturn(anOkResponse().withBody(UnitLinksForTargetUbrnWithoutChildVatHBaseResponseBody)))
+
+      When(s"the deletion of the child link from $TargetUBRN to the VAT unit with reference $VatRef is requested")
+      val response = await(wsClient.url(s"/v1/periods/${Period.asString(RegisterPeriod)}/types/$LegalUnitAcronym/units/$TargetUBRN").
+        withHeaders(CONTENT_TYPE -> JsonPatchMediaType).
+        patch(s"""[{"op": "test", "path": "/children/$VatRef", "value": "$VatUnitAcronym"},
+                  |{"op": "remove", "path": "/children/$VatRef"}]""".stripMargin))
+
+      Then("a Success response is returned")
+      response.status shouldBe NO_CONTENT
+    }
+
+    /*
+     * The target row does not exist.
+     */
+    scenario("when the Legal Unit does not exist in the register") { wsClient =>
+      Given(s"there does not exist a Legal Unit identified by UBRN $TargetUBRN")
+      stubHBaseFor(aUnitLinksExactRowKeyRequest(withUnitId = TargetLegalUnitId, withUnitType = LegalUnit, withPeriod = RegisterPeriod).
+        willReturn(anOkResponse().withBody(NoMatchFoundResponse)))
+
+      When(s"the deletion of the child link from $TargetUBRN to the VAT unit with reference $VatRef is requested")
+      val response = await(wsClient.url(s"/v1/periods/${Period.asString(RegisterPeriod)}/types/$LegalUnitAcronym/units/$TargetUBRN").
+        withHeaders(CONTENT_TYPE -> JsonPatchMediaType).
+        patch(s"""[{"op": "test", "path": "/children/$VatRef", "value": "$VatUnitAcronym"},
+                  |{"op": "remove", "path": "/children/$VatRef"}]""".stripMargin))
+
+      Then(s"a Not Found response is returned")
+      response.status shouldBe NOT_FOUND
+    }
+
+    scenario("when the specification of the deletion is valid but inappropriate for deleting a link to a child VAT unit") { wsClient =>
```

Comment:
Is this due to an incorrect `unitType`, where `$CompaniesHouseAcronym` should be a VAT acronym? It may be useful to have this reflected in a `Given`/`And` somewhere.